### DISCUSSION TITLE
Changing the workspace label by adding a line to Xresources doesn't work

### DISCUSSION THREAD

**Discussion by oren** [regolith-linux/regolith-desktop#1053] [Q&A]
I am trying to change the workspace labels as described here: https://regolith-desktop.com/docs/howtos/change-workspace-icons/

Steps to reproduce:
I added 3 lines to ~/.config/regolith3/Xresources:
```
wm.workspace.01.name: 1: Terminal
wm.workspace.02.name: 2: Web
wm.workspace.03.name: 3: Chat
```
In a new terminal, I run `regolith-look refresh`.

But the labels remains the same.

**Comment by w18g** [comment]
I don't think the Xresources will rename an existing workspace. You will have to delete that workspace and recreate it (i.e in your example recreate 1, 2, 3). Or just log out and log back in.

**Comment by oren** [comment]
Thank you! You are correct
